Coding/fw/lib/Lawicel/Lawicel.h

```diff
@@ -0,0 +1,157 @@
+/***************************************************************************************************
+  (c) NewTec GmbH 2020   -   www.newtec.de
+  $URL: https://github.com/NewTec-GmbH/esp32-can-iot $
+***************************************************************************************************/
+/**
+@addtogroup Lawicel
+@{
+@file       Lawicel.h
+
+Driver for Lawicel Protocol
+
+* @}
+***************************************************************************************************/
+#ifndef LAWICEL_H_
+#define LAWICEL_H_
+
+/* INCLUDES ***************************************************************************************/
+#include <Arduino.h>
+#include <string.h>
+#include <CANInterface.h>
+#include <SerialInterface.h>
+#include <NVMInterface.h>
+
+#define MAX_TIMESTAMP 0xEA5F    /**< Maximum value of a Timestamp, representing 1 Minute. */
+#define X_VERSION "V0101"       /**< Hardware and Software Version. */
+#define X_SERIAL_NUMBER "NNT32" /**< Hardware Serial Number. */
+#define CR 13                   /**< CR Character as OK Line Terminator */
+#define BELL 7                  /**< BEL Character as ERROR Line Terminator */
+
+/**
+* Key Definition (Adresses) to be read by NVM Adapter for initialization
+*/
+#define INIT_SERIAL_BAUD "SerialBaud"
```

Comment:
#defines for strings  are bad as the may consume memory each time they asre used unless linker magic like string pooling is active. Convert to constants defined in header, but declared in a cpp